### ISSUE TITLE
No accessible name "Keep up with our latest.." link

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#75]
This element has no HTML markup that would provide an accessible name for the element. A screen reader would either read nothing for this element or may make a guess.
